account/client_test.go

```diff
@@ -56,11 +56,16 @@ func TestAccountNew(t *testing.T) {
 			AdditionalOwners: []*stripe.AdditionalOwnerParams{
 				{
 					FirstName: stripe.String("Jane"),
+					LastName:  stripe.String("Doe"),
 					Verification: &stripe.IdentityVerificationParams{
 						Document:     stripe.String("file_345"),
 						DocumentBack: stripe.String("file_567"),
 					},
 				},
+				{
+					FirstName: stripe.String("John"),
+					LastName:  stripe.String("Doe"),
+				},
```

Comment:
Added a little more flavor to this test just so that I could see multiple array elements being encoded for myself.